apps/mull-ui/src/app/pages/home/my-events/my-events.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { IEvent } from '@mull/types';
+import { EventCard } from '../../../components';
+import { gql, useQuery } from '@apollo/client';
+
+import '../home-discover.scss';
+
+interface MyEventData {
+  coHostEvents: Partial<IEvent>[];
+  hostEvents: Partial<IEvent>[];
+}
+
+export const GET_PARTICIPATING_EVENTS = gql`
+  query getUsersEvents($UserId: Int!) {
+    coHostEvents(userId: $UserId) {
+      id
+      title
+      description
+      startDate
+      endDate
+    }
+    hostEvents(userId: $UserId) {
+      id
+      title
+      description
+      startDate
+      endDate
+    }
+  }
+`;
+
+export const MyEventsPage = () => {
+  const { data } = useQuery<MyEventData>(GET_PARTICIPATING_EVENTS, {
+    variables: { UserId: 1 },
```

Comment:
Make sure to leave a `//TODO` for this, and all other places where the user id is hardcoded right now